tests/typerel/trectuple.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   errormsg: "illegal recursion in type 'TNode'"
   line: 8
-  disabled: true
+    disabled: true
```

Comment:
Disabled around 2014 in a commit that 'tuple field names are ignored'

Reply:
Does it work?

Reply:
Currently errors, but i believe its easily fixable:

```Failure: reLinesDiffer
Expected:
8:0

Gotten:
9:13
```

Reply:
Does 8:0 even make sense, not saying 9:13 is much better, maybe slightly... they're both silly, just degrees.

I'd say update the line number and bring it back?